templates/hooks/open_id_connect_provider/sdk_update_post_set_output.go.tpl

```diff
@@ -0,0 +1,7 @@
+    if err := rm.syncTags(ctx, &resource{ko}); err != nil {
+        return nil, err
+    }
+    // There really isn't a status of an OIDC Provider... it either exists or doesn't. If
+    // we get here, that means the update was successful and the desired state
+    // of the OIDC Provider matches what we provided...
+    ackcondition.SetSynced(&resource{ko}, corev1.ConditionTrue, nil, nil)
```

Comment:
You can't use `sdk_update_*` hooks with a custom update method, since we expect you'd just hardcode the method entirely and circumvent the need to inject code. This can be removed

Reply:
Done